utils/index.spec.js

```diff
@@ -51,6 +59,33 @@ describe('Test hasData function', () => {
   test('returns true for disabilityClaim', () => {
     expect(hasData(user, 'deductions.disabilityClaim')).toBe(false)
   })
```

Comment:
Let's write a test for `hasData` so that we see it returns the value, not just true or false.

Reply:
Oh wait, you removed the logic. Okay, I'm getting rid of the extra param in hasData.